### ISSUE TITLE
Aporte y ayuda

### ISSUE THREAD

**Issue by Roberto-LP** [ucutumbu/Aplicacion_Aerolineas#1]
Hola, buenas. Me gustaría contribuir a tu proyecto agregando algunas funciones que faltaban. ¿Podrías revisarlas y decirme qué te parecen? Además, agradecería tu ayuda para mi progreso de aprendizaje y algunos consejos para mejorar.

**Comment by ucutumbu** [comment]
Claro, no hay problema. Solo recuerda crear una nueva rama para tus cambios y luego envíame tus sugerencias. Así podré revisarlas más tarde.
